platform/ios/jazzy.yml

```diff
@@ -23,55 +26,62 @@ custom_categories:
       - MGLMapViewDelegate
       - MGLStyle
       - MGLUserTrackingMode
-  - name: Annotations
+  - name: Protocols and Abstract Classes
```

Comment:
OK, let’s move them back then. I’ll also work on a way for a way to provide more info about each class in the table of contents within jazzy.yml so we don’t have to hard-code that list inside SCSS. 😬